packages/core/core/src/RequestTracker.js

```diff
@@ -572,15 +592,15 @@ export class RequestGraph extends Graph<
 
           if (above.length > 0) {
             didInvalidate = true;
-            this.invalidateFileNameNode(fileNameNode, filePath, above);
+            this.invalidateFileNameNode(fileNameNode, _filePath, above);
           }
         }
 
         for (let id of this.globNodeIds) {
           let globNode = this.getNode(id);
           invariant(globNode && globNode.type === 'glob');
 
-          if (isGlobMatch(filePath, globNode.value)) {
+          if (isGlobMatch(filePath, fromProjectPathRelative(globNode.value))) {
```

Comment:
So `filePath` here is absolute, but the glob is relative? Won't that not match except if the glob starts with `**/`? I was thinking about how to do this for the glob resolver. Should the glob in the invalidation be relative to the project root? If so, then I think we'd need to make it absolute here.

Reply:
`filePath` is also relative here, further up there's

```js
let filePath = fromProjectPathRelative(_filePath);
```

Reply:
But I ran into https://github.com/parcel-bundler/parcel/pull/5483#issuecomment-746479825 again, so I guess I'll have to use the absolute paths anyway (to make micromatch happy).

As a workaround, I've downgraded to micromatch 3 for now 😬